src/tests/test_blindsig.py

```diff
@@ -6,12 +6,12 @@
 from hashlib import sha256
 from .common import skip_python3
 
-skip_python3()
-
 from pybitmessage.pyelliptic import ECCBlind, ECCBlindChain, OpenSSL
 
 # pylint: disable=protected-access
 
+skip_python3()
+
```

Comment:
there is no `skip_python3()` in v0.6, this hunk is not fit

Reply:
Yep, Fixed now